10up/ElasticPressLabs
# Update User indexable and remove old mapping

### Description of the Change
This PR will update the User Indexable and User mapping files with the latest changes from the main plugin. It will also removes the old mapping files. 



Closes #70 

### How to test the Change
- Install the plugin and enable the Users feature
- Make sure that the Users feature is working properly
- Make sure that the new filters that were added are working fine.


### Changelog Entry

> Changed - Update the User Indexable files
> Removed - Remove Old Mapping files for Users

### Credits

Props @MARQAS 

### Checklist:
